.htmltest.yml

```diff
@@ -13,6 +13,11 @@ IgnoreURLs:
   - "https://crates.io/*"
   - "https://docs.pipenv.org/*"
   - "https://arxiv.org/*"
+  - "https://www.microsoft.com/*"
+  - "https://v1-18.docs.kubernetes.io/*"
```

Comment:
This will get rid of the errors, but the links are still broken.

Reply:
You're right, but @taroface said he's going to do work into decoupling the K8s docs with our docs.

Ryan, is there a JIRA ticket for that? If not, do you want me to create one?